client/db.json

```diff
@@ -20,6 +20,13 @@
       "date": "26.12.2020",
       "time": "13:30",
       "distance": "7,5"
+    },
+    {
+      "runName": "Aachener Weiher Run",
+      "id": 4,
+      "date": "27.12.2020",
```

Comment:
Not important for now, but later, you should save dates in ISO-format or JSON (or in Date if you are using MongoDB).
This makes it easier to convert it to a date object and compare it to other dates.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toISOString
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toJSON

Reply:
Alright. Thanks for the resources. 
I`ll make this a new issue. :)